### ISSUE TITLE
Developing AGE Viewer desktop application in Python 

### ISSUE THREAD

**Issue by aked21** [apache/age-viewer#72]

**Comment by Nimra-1234** [comment]
Yes, it's a good idea to develop AGE Viewer desktop application using Python as:

Python is a high-level, object-oriented programming language that eliminates the need for a programmer to write longer lines of code to run a concept. It is one of the most adaptable languages available. Not only that, but Python has a wide variety of GUI frameworks that allow developers to easily and securely create applications. Some of the frameworks that you can use with Python to develop not only Windows applications but also other applications include, PyQT (a combination of the QT library and Python) allows the developer to choose between creating visual dialogues with QT Designer and writing code. Kivy (an OpenGL ES2 framework) allows programmers to write code once and run it across multiple platforms; WxPython, PyGUI, and Tkinter.

We can have so many options for implementing many functions without a lot of hardwork, as we have many built-in functions present in it.

**Comment by saadjameel** [comment]
Yes I have started setting up the development environment in python pyQT. A large number of built-in libraries of python may increase it's efficiency and make it more reliable.

**Comment by sarthak-19** [comment]
Age Viewer desktop application using Golang has been finalized so I think this issue should be closed !
